week-2-practise/level-2/6-map.js

```diff
@@ -4,7 +4,10 @@
 //    - each item in `percentages` formatted by your function
 
 var percentages = [1, 23, 92, 18];
-
+function strWithPercentage(number) {
+  return number + "%";
```

Comment:
It's always handy to remember that using the `+` operator on any type plus a string, will give a string output e.g.
```
var x = true + "test";
x; // "truetest"
```